meta-mender-tegra/recipes-bsp/tegra-binaries/redundant-boot-overrides/update-nvbootctrl.service.in

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Update bootloader on successful boot
+Wants=mender-client.service
+After=mender-client.service
```

Comment:
@madisongh one thing we've noticed with automated testing:  When ssh login/reboot happens before the update-nvbootctrl.service starts (regardless of whether a mender update has occurred on the previous boot) you can get into a state where the boot slot changes between normal reboot cycles.  If I add a 15 second delay after the system is first available over ssh I was able to go 100 reboots without issue.  Our guess is it's because of the delay in starting this service but I haven't proven this yet.  I'm wondering if it would be better to make this before `network-pre.target` to ensure reboots are safe to do as soon as the device is up on the network.  Ideally I supposed we would keep an `After=mender-client.service` for the case where you performed a mender update on the previous boot, but this might be tricky to figure out.

Reply:
Yeah, this might be something that we'd want to make configurable, as the definition of a successful boot may vary from use case to use case.  In my production builds at work, for instance, I defer the marking of successful boot until after every other service has been started, since a failure of any of those services could mean that the image is no good.

But maybe it would be better to default to running this earlier rather than later by default, and let folks use a drop-in if they need to defer it.

Reply:
I updated the service unit file to change the ordering to just `Before=network.target`.